### ISSUE TITLE
Tailwind build depends on html in template folder. Changes not being detected.

### ISSUE THREAD

**Issue by kingkong-cmd** [miracle2k/flask-assets#148]
I am struggling to trigger rebuild of tailwind css bundle whenever html files changes, using the 'depends'-variable. I believe it may be due to a bad specification of the path to my template folder (templates). Is my path to the templates folder correctly specified?

Folder structure:
```
app
|-->static
   |-->tailwind
   |-->gen_tailwind
|-->templates
```

Bundle initialization:
```
css_tailwind = Bundle("tailwind/main.css",
             filters="postcss", 
             output="gen_tailwind/tailwind.css",
             depends='../templates/tailwind/**/*.html')
```

**Comment by bitstein** [comment]
I have had the same problem. Your path is correctly specified. I believe there is a problem with how the glob patterns are being processed, specifically the recursion (`**/`). If I don't use that, it works, but then I have to specify all directories.

**Comment by klassisti** [comment]
This is a `webassets` issue; the path resolver was probably written before the stdlib `glob` supported recursive patterns, and it hasn't been updated accordingly. It can be made to work by installing `glob2` (which `webassets` will look for but doesn't ship with) or by overriding/monkey-patching the resolver's `glob` method to support recursive globbing.

**Comment by kingkong-cmd** [comment]
Thanks @klassisti. 

If anyone else is having related issues, I circumvented the problem in its entirety. Tailwind now enables build from a standalone cli, and I used this python wrapper (https://github.com/timonweb/pytailwindcss) to integrate the cli into my Flask app. Now the Tailwind css file is built in real-time without involving flask-assets or any other js/node framework involvement in the css build process. 

The cli is a life saver for anyone using tailwind with python and hate having to expand the stack with Node stuff for the sole purpose of using Tailwind.
